From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Todd Kjos <tkjos@google.com>
Date: Wed, 4 Mar 2020 13:39:38 -0800
Subject: ANDROID: gki_defconfig: disable KPROBES

Security teams from Google and Qualcomm request disabling
due to "large attack vector".

Bug: 149659981
Signed-off-by: Todd Kjos <tkjos@google.com>
Change-Id: Ie0a8e835bca8c7111f74440b1a44680856aae182
---
 arch/arm64/configs/gki_defconfig | 1 -
 arch/x86/configs/gki_defconfig   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -76,7 +76,6 @@ CONFIG_ARM_SCPI_PROTOCOL=y
 CONFIG_ARM64_CRYPTO=y
 CONFIG_CRYPTO_SHA2_ARM64_CE=y
 CONFIG_CRYPTO_AES_ARM64_CE_BLK=y
-CONFIG_KPROBES=y
 CONFIG_SHADOW_CALL_STACK=y
 CONFIG_LTO_CLANG=y
 CONFIG_CFI_CLANG=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -55,7 +55,6 @@ CONFIG_CPU_FREQ_DEFAULT_GOV_SCHEDUTIL=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_IA32_EMULATION=y
-CONFIG_KPROBES=y
 CONFIG_LTO_CLANG=y
 CONFIG_CFI_CLANG=y
 CONFIG_MODULES=y
